webpack.config.babel.js

```diff
@@ -6,11 +6,14 @@ const rootPath = path.resolve(__dirname, "./");
 const srcPath = path.resolve(rootPath, "./src");
 
 const HOST = process.env.HOST || "localhost";
-const { BM_RPC_ADDR } = process.env;
+const { BM_RPC_ADDR, CUSDEB_API_URL } = process.env;
 
 if (!BM_RPC_ADDR) {
     throw new Error("The Black Magic RPC server address is not specified.");
 }
+if (!CUSDEB_API_URL) {
+    throw new Error("The cusdeb-api server address is not specified.");
```

Comment:
Let's substitute `cusdeb-api` for `CusDeb API` (by analogy with `Black Magic RPC`).